DATA/SConscript

```diff
@@ -241,6 +248,22 @@ defects = env.Command([os.path.join(REPO_ROOT, 'ci_cpp_defects'),
                        getCertifyCmd('defects')])
 env.Alias('defects', defects)
 
+# Create DEFECTS using combined darks and flats.
+defectsCombined = env.Command([os.path.join(REPO_ROOT, 'ci_cpp_defectsCombined'),
+                       os.path.join(REPO_ROOT, "calib", 'v00', "defectsCombined")],
+                       # This only really depends on crosstalk, but we don't
+                       # want scons to run multiple pipelines at a time, since
+                       # database, so we pretend it depends on gain as well.
```

Comment:
"since database"?

Reply:
Anyway, it's fine to say "while this pipeline doesn't actually depend on all of these, we declare these dependencies to scons to avoid database contention."

Reply:
This was copy-paste from "defects" so I don't know what it means. I'm pining @czwa in case he has any comments, but I'll just replace the full comment with what you suggest.